Cargo.toml

```diff
@@ -30,7 +30,7 @@ reqwest = "0"
 colored = "2"
 lame = "0"
 aspotify = "0"
-librespot = { git = "https://github.com/librespot-org/librespot", rev = "f647331" }
+librespot = "0"
```

Comment:
Sorry for not seeing this earlier! Looks nice overall, I'll give it a try - haven't used it in ages. I switched to git because the published library had broken, but I see they've updated the published package since. Due to the volatility of the matter at hand, it might be a good idea to pin this to a specific version so it's easier to track breaking changes from upstream dependencies.